src/QirRuntime/test/QIR-static/qir-gen.ll

```diff
@@ -1,1690 +1,1691 @@
-%Result = type opaque
```

Comment:
I thought this file was deleted by my change in #522 which should have updated the gitignore too. How is it ending up in the diff here?

Reply:
Good question. Don't know. I deleted the file and rebuilt locally again, and it was correctly filtered out by .gitignore this time... Thanks for catching this!

Oh, I see! @kuzminrobin change https://github.com/microsoft/qsharp-runtime/commit/ec763fbeb513b85beebc9985ae8e0d40162f573c added the file back. Likely, because it was updated compared to the pre-delete state and merge decided to resurrect it... I guess, we'll need to keep an eye on this for a while.

Reply:
Looks like during rebasing my branch, I resolved the conflict incorrectly such that the file was recovered.
I assume that now this file needs to be always generated (if we build the tests in a clean directory).